src/Microsoft.Health.Dicom.Core/Features/Export/Input/ExportAzureStorage.cs

```diff
@@ -0,0 +1,12 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.Export;
+public class ExportAzureStorage
+{
+    public string StorageAccountName { get; set; }
```

Comment:
Having thought a little about that, having the URI is better, since it allows the user to specify a directory path. and we can write the dcm files relative to a directory

Reply:
this file is removed.